### PR TITLE
chore(nextcloud): Update specs to 28.0.1

### PR DESCRIPTION
**Pull request by provokateurin**
Closes https://github.com/nextcloud/neon/issues/1226

Draft for now, I'll wait until 28.0.1 has been tagged on server and notifications (although there are no diffs using the stable28 branches).

### CODE REVIEW

File: packages/nextcloud/lib/src/api/files_sharing.openapi.json [hunk 1]
```diff
@@ -2793,26 +2801,20 @@
...
-                                        "type": "array",
-                                        "items": {
-                                            "type": "integer",
-                                            "format": "int64"
-                                        }
                                     }
-                                ]
-                            }
+                                }
+                            ]
```
Comment (Leptopoda) [inline packages/nextcloud/lib/src/api/files_sharing.openapi.json:2817]: I'd like a unit test on that one.
It should not be affected by #1058 but I'd like to have it tested.

File: packages/nextcloud/lib/src/api/files_sharing.openapi.json [hunk 2]
```diff
@@ -2916,26 +2918,20 @@
...
```
Comment (Leptopoda) [inline packages/nextcloud/lib/src/api/files_sharing.openapi.json:2934]: same here

**Review by Leptopoda — COMMENTED** [review]
LGTM.

The unit tests do not need to be in NC if it is hard to reproduce both available cases.
Maybe add a dynamite_e2e test with `oneof(int|array<int>)` and `oneof(String|array<Object>)` (the later one being used in '/ocs/v2.php/apps/spreed/api/{apiVersion}/signaling/{token}'

**Comment by provokateurin** [comment]
The spreed case of `oneof(String|array<Object>)` is already tested. Seems like oneOf with array is completely broken :/ I'm opening a separate PR with a test so we can merge this already.

**Comment by provokateurin** [comment]
I think we can merge this already because the bug is in dynamite and doesn't block this change.

File: packages/nextcloud/lib/src/api/files_sharing.openapi.json (hunk 1 above)
Comment (provokateurin) [reply packages/nextcloud/lib/src/api/files_sharing.openapi.json:2817]: Nope this is exactly https://github.com/nextcloud/neon/pull/1215 :/

**Review by Leptopoda — APPROVED** [review]

**Pull request merged**